includes/controllers/class-attachment.php

```diff
@@ -266,6 +266,13 @@ public function upload_attachment( $request ) {
 			)->save( [ $parent_field->get_name() ] );
 		} else {
 
+			// Update parent object.
+			$parent->fill(
+				[
+					$parent_field->get_name() => array_merge( (array) call_user_func( [ $parent, 'get_' . $parent_field->get_name() ] ), [ $attachment->get_id() ] ),
```

Comment:
Please explain the solution, I see that a single ID is saved in the "parent" field, or the task is waiting for clarification (based on the recent Asana comment)? If so then please let me know and re-assign the task back, I'll pre-check the solution in detail and leave some guidelines

Reply:
There was a problem that attachment IDs are not saved for the field when multiple option is enabled for attachment field. Currently, multiple option is enabled only for images listing model field and pretty simple code works only for images listing model field in action hook models/listings/update_images